src/styles/main.scss

```diff
@@ -1,3 +1,235 @@
-body {
+$blue_color: #00acdc;
+$dark-blue_color: #060b35;
+$card-items_width: 166px;
+
+@mixin card_size {
+  width: 198px;
+  height: 406px;
+}
+
+@mixin star_size {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.header,
+.nav__list,
+.nav__item,
+.nav__link,
+.is-active {
+  display: flex;
+}
```

Comment:
it's better to add display: flex; separately to each element

Reply:
done